### PR TITLE
feat: bootstrap support run on jest env

### PR DESCRIPTION
**Pull request by fengmk2**
before => beforeAll

##### Checklist

- [ ] `npm test` passes
- [ ] tests and/or benchmarks are included
- [ ] documentation is changed or added
- [ ] commit message follows commit guidelines

### CODE REVIEW

File: .eslintrc [hunk 1]
```diff
@@ -1,3 +1,6 @@
 {
-  "extends": "eslint-config-egg"
+  "extends": "eslint-config-egg",
+  "globals": {
+    "beforeAll": true
```
Comment (dead-horse) [inline .eslintrc]: 这些 jest 相关的要配置到 eslint-config-egg 中吧

Comment (atian25) [reply .eslintrc]: 要的

**Comment by atian25** [comment]
CI 的报错，在 https://github.com/eggjs/egg-mock/pull/92/files#r238549847 里面有修复了

**Comment by atian25** [comment]
3.21.0

**Pull request merged**